testsuite/modern-testsuite/src/test/it-resources/org/sonatype/nexus/testsuite/maven/testplugin/src/main/java/org/sonatype/nexus/testsuite/MyMojo.java

```diff
@@ -0,0 +1,84 @@
+package org.sonatype.nexus.testsuite;
+
+/*
+ * Copyright 2001-2005 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Goal which touches a timestamp file.
+ *
+ * @deprecated Don't use!
+ */
+@Mojo( name = "touch", defaultPhase = LifecyclePhase.PROCESS_SOURCES )
```

Comment:
This s maven quickstart archetype as-is, and basically we don't care about it's contents, we want maven to deploy it... I think it's ok to leave it as is, maybe a README in root to describe from where these sources come (maven-quickstart-archetype:1.1 I think)